gradle/libs.versions.toml

```diff
@@ -19,7 +19,7 @@ appcompat = "1.4.1"
 dokka = "1.8.10"
 androidxCore = "1.8.0"
 dependencyGraph = "0.9"
-kover = "0.7.0"
+kover = "0.7.2"
```

Comment:
Updated to kover 0.7.2 which has the fix for issue that forced us to reference the amazon module in the purchases module.